docker-compose.yml

```diff
@@ -8,5 +8,5 @@ services:
         volumes:
         # HOST : CONTAINER
         - "./.env:/app/.env"
-        - "/data/application-data/speech-db/db.sqlite3:/app/db.sqlite3"
-        - "/data/application-data/speech-db/audio:/app/data/audio"
\ No newline at end of file
+        - "./db.sqlite3:/app/db.sqlite3"
```

Comment:
If this isn't a mounted volume anymore, it likely doesn't need to be in here, right?

Reply:
We definitely need this; otherwise the container won’t have a database to work with on startup, and if you do manage to get some data saved into the container, it’ll be lost when a new container is created on the next deploy.

Mounting a file or directory into a container is similar but different from mounting a network share. `/data` on itw/kor is a network share mounted from the gw machine. When only a docker mount is involved, the process is much more reliable than going over a network, but you still run into some issues that you wouldn’t see if no extra levels of mounting were involved, e.g., some of the stuff we see with UIDs and permissions.

Reply:
Oh of course, I didn't see the `:` when I made the comment